test/gui/shared/scripts/pageObjects/SharingDialog.py

```diff
@@ -82,3 +88,20 @@ def addCollaborator(self, receiver, permissions):
 
     def getErrorText(self):
         return str(squish.waitForObjectExists(self.ERROR_SHOWN_ON_SHARING_DIALOG).text)
+
+    def removePermissions(self, receiver, permissions):
+        removePermissionsList = permissions.split(",")
+        (
+            isEditPermissionAvailable,
+            isSharePermissionAvailable,
+        ) = self.getAvailablePermission()
+
+        if 'share' in removePermissionsList and isSharePermissionAvailable == True:
```

Comment:
```suggestion
        if 'share' in removePermissionsList and isSharePermissionAvailable:
```
We do not need to compare if the variable is already a boolean.